src/components/pages/PointShare/usePointShare.js

```diff
@@ -36,7 +36,7 @@ const usePointShare = () => {
   const calculatePointsLeft = () => {
     return totalPoints - pointsUsed();
   };
-
+// eslint-disable-next-line
   const pointsLeft = useMemo(() => calculatePointsLeft(), [points]);
```

Comment:
same thing with this, add dependency > disabling linting. Does adding the dependency cause issues?

Reply:
hmm adding it gave the following error 👍 
'React Hook useMemo has an unnecessary dependency: 'points'. Either exclude it or remove the dependency array'

as above I think I will just leave a todo comment on this one for the future team to come in and audit.